src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -2366,6 +2401,45 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 break;
             }
 
+            case NI_Sve_LoadVectorFirstFaulting:
+            {
+                assert(op3Reg == REG_NA);
```

Comment:
Just in case, lately the check began to fail on my machine:

```
/home/mikabl01/dotnet/runtime/artifacts/tests/coreclr/linux.arm64.Checked/Tests/Core_Root/corerun -p System.Reflection.Metadata.MetadataUpdater.IsSupported=false -p System.Runtime.Serialization.EnableUnsafeBinaryFormatterSerialization=true HardwareIntrinsics_Arm_ro.dll 'LoadVectorFirstFaulting'
17:56:34.401 Running test: _Sve_ro::JIT.HardwareIntrinsics.Arm._Sve.Program.Sve_LoadVectorFirstFaulting_float()
Supported ISAs:
  AdvSimd:   True
  Aes:       True
  ArmBase:   True
  Crc32:     True
  Dp:        True
  Rdm:       True
  Sha1:      True
  Sha256:    True
  Sve:       True

Beginning scenario: RunBasicScenario_Load

Assert failure(PID 2387565 [0x00246e6d], Thread: 2387565 [0x246e6d]): Assertion failed 'op3Reg == REG_NA' in 'JIT.HardwareIntrinsics.Arm._Sve.Sve__Sve_LoadVectorFirstFaulting_float:RunBasicScenario_LoadFirstFaulting():this' during 'Generate code' (IL size 138; hash 0x10c6e74d; Tier0)

    File: /home/mikabl01/dotnet/runtime/src/coreclr/jit/hwintrinsiccodegenarm64.cpp:2406
    Image: /home/mikabl01/dotnet/runtime/artifacts/tests/coreclr/linux.arm64.Checked/Tests/Core_Root/corerun

./artifacts/tests/coreclr/linux.arm64.Checked/JIT/HardwareIntrinsics/HardwareIntrinsics_Arm_ro/HardwareIntrinsics_Arm_ro.sh: line 432: 2387565 Aborted                 (core dumped) $LAUNCHER $ExePath "${CLRTestExecutionArguments[@]}"

```

Reply:
yes, this will be fixed as i pointed in https://github.com/dotnet/runtime/pull/105348#discussion_r1688787121